MHArmory.Search/OpenCL/Host.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.ObjectModel;
+using System.Text;
+using Cloo;
+
+namespace MHArmory.Search.OpenCL
+{
+    class Host : IDisposable
+    {
+        private ComputeContext Context { get; }
```

Comment:
Please change private properties to private fields with camel case naming, such as:
```CS
private ComputeContext context;
private ComputeProgram program;
```